.pre-commit-config.yaml

```diff
@@ -15,4 +15,5 @@ repos:
     hooks:
     -   id: flake8
         name: flake8
-        entry: cd api && flake8
+        entry: bash -c 'git diff --staged -- "$@" | flake8 --diff' --
```

Comment:
Wouldn't this need the config file moved / specified too?

Reply:
it reads the config the root path - hence why the file moved. This code also only checks changed files - doing the full repo took like 6 second which would be really annoying.